service-onboarding-build-pack/Jenkinsfile

```diff
@@ -30,49 +31,64 @@ def Event_Status = [
 	'COMPLETED':'COMPLETED',
 	'FAILED':'FAILED'
 ]
-@Field def region = ''
+@Field def region  = ''
+@Field def scm_base = '' 
 
-node  {
-	region = env.jazz_region
-	def service_type =  params.service_type
-	def runtime = params.runtime
-	def service_name = params.service_name
-	def admin_group = "admin_group"
-	def bitbucket_name = params.service_name
-	
+node  {	
 	def service_template
 	def service_username
 	def service_password
-	def domain = params.domain
-	def owner = ""
-	if (params.username) {
-		owner = params.username
-	}
-
+	
+	def service_type   				= params.service_type
+	def runtime        				= params.runtime
+	def service_name   				= params.service_name
+	def admin_group    				= "admin_group"
+	def bitbucket_name 				= params.service_name
+	def domain		                = params.domain
+
+	def scm				            = env.JAZZ_SCM
+	def owner  	                    = ""
 	def repo_protocol				= "http://"
-
 	def var_credentialsId 			= env.REPO_CREDENTIAL_ID
 	def var_bitbucket_base			= env.REPO_BASE
-	def var_bitbucket_framework_url = repo_protocol + var_bitbucket_base + "/scm/" + env.REPO_CORE + "/"
 	def var_api_key 				= env.API_KEY
 	def var_api_host_name			= var_api_key+".execute-api.us-east-2.amazonaws.com"
 	def var_cas_rest_repo 			= repo_protocol + var_bitbucket_base  + "/rest/api/1.0/projects/" + env.REPO_LOC + "/repos"
 	def var_bitbucket_email 		= env.REPO_EMAIL
-	def var_cas_clone_url	 		= var_bitbucket_base + "/scm/" + env.REPO_LOC + "/"
-	def var_serviceonboarding_repo 	= repo_protocol + var_bitbucket_base + "/scm/" + env.REPO_CORE + "/service-onboarding-build-pack.git"
+
 	def var_bitbucket_branch_perm	= repo_protocol + var_bitbucket_base + "/rest/branch-permissions/2.0/projects/" + env.REPO_LOC + "/repos/"
 	def var_bitbucket_webhook_api	= repo_protocol + var_bitbucket_base + "/rest/webhook/1.0/projects/" + env.REPO_LOC + "/repos/"
-	def var_scm_source_hook 		= JenkinsLocationConfiguration.get().getUrl()+"/bitbucket-scmsource-hook/notify"
+	def var_jenkins_url             = JenkinsLocationConfiguration.get().getUrl()
```

Comment:
for consistency call this jenkins_url (atleast the new variables we should try to remove the var in the front)